src/ape_plugins/utils.py

```diff
@@ -373,14 +373,27 @@ def __bool__(self) -> bool:
         return len(self.plugins) > 0
 
     def __repr__(self) -> str:
-        return f"<{self.name} Plugins Group>"
+        try:
+            return f"<{self.name} Plugins Group>"
+        except Exception:
+            # Prevent exceptions happening in repr()
+            logger.log_debug_stack_trace()
+            return "<PluginGroup>"
 
     def __str__(self) -> str:
         return self.to_str()
 
+    @validator("plugin_type")
+    def validate_plugin_type(cls, value):
+        return PluginType(value) if isinstance(value, str) else value
```

Comment:
question: So was something not initializing with a valid enum?

Reply:
somehow yes but I don't see where! I can't find he instance where we do that anywhere in our code, that's why im quite confused... it seems like sometimes the `Enum` part doesnt work in pydantic.